src/app/containers/Disclaimer/index.jsx

```diff
@@ -51,7 +52,18 @@ const DisclaimerComponent = ({ increasePaddingOnDesktop }) => {
         script={script}
         increasePaddingOnDesktop={increasePaddingOnDesktop}
       >
-        {path(['text'], disclaimer)}
+        {disclaimerBlock.map(element => {
+          const isLink = pathOr(false, ['link'], element);
+          const linkText = pathOr('', ['link', 'text'], element);
+          const linkHref = pathOr('', ['link', 'href'], element);
+          return isLink ? (
+            <InlineLink href={linkHref} key={linkText}>
+              {linkText}
+            </InlineLink>
+          ) : (
+            element?.text
```

Comment:
Are there any console errors being output due to this not having a `key`?

Reply:
Not that I can see :D, I believe react sees that as text only